src/toil/utils/toilDebugJob.py

```diff
@@ -60,6 +60,6 @@ def main():
 
     # Run the job locally
     jobID = options.jobID[0]
-    logger.info("Going to run the following job locally: %s", jobID)
+    logger.debug("Going to run the following job locally: %s", jobID)
     workerScript(jobStore, config, jobID, jobID, redirectOutputToLogFile=False)
-    logger.info("Ran the following job locally: %s", jobID)
\ No newline at end of file
+    logger.debug("Ran the following job locally: %s", jobID)
```

Comment:
very nitpicky, but github is complaining about the lack of newline at end of file